lang/en/texts/google-summer-of-code.html

```diff
@@ -1,6 +1,6 @@
 <!-- no side column -->
 
-<h1>Open Food Facts - Google Summer of Code (GSOC) 2023</h1>
+<h1>Open Food Facts - Google Summer of Code (GSOC)</h1>
```

Comment:
```suggestion
<h1>Open Food Facts - Google Summer of Code (GSOC) 2024</h1>
```

Maybe it's a good practice to make sure the document is updated each year, to appapt to GSoC rule modification

Reply:
Huuum, I was more on the verge of avoiding this page to decay ;-)

Those are most best practice that we decided.